docs/sources/flow/reference/components/discovery.gce.md

```diff
@@ -0,0 +1,78 @@
+---
+title: discovery.gce
+---
+
+# discovery.gce
+
+`discovery.gce` allows retrieving scrape targets from [GCP GCE](https://cloud.google.com/compute) instances. The private IP address is used by default, but may be changed to the public IP address with relabeling.
+
+
+## Usage
+
+```river
+discovery.gce "LABEL" {
```

Comment:
The usage snippet should show required arguments/blocks:

```suggestion
discovery.gce "LABEL" {
  project = "PROJECT_NAME" 
  zone    = "ZONE_NAME"
```

Reply:
Ah yeah, i forgot :sweat_smile: